docs/framework/windows-workflow-foundation/wfc-exe-workflow-compiler-tool.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Wfc.exe (Workflow Command-line Compiler Tool)"
+description: Understand wfc.exe, the Workflow command line compiler tool. Use the tool to generate code and mapping for the LINQ to SQL component of .NET.
+ms.date: "10/10/2020"
+helpviewer_keywords: 
+  - "wfc [Workflow]"
+  - "compiler tool"
+  - "wfc.exe"
+  - "Workflow, compilation"
+  - "Workflow, XOML files"
+  - "Workflow, wcf"
+ms.assetid: 7d2bf3a5-6624-4a8e-b546-321d36b757d4
+---
+# wfc.exe (Workflow Command-line Compiler Tool)
+> [!NOTE]
+>  [!INCLUDE[DeprecatedContent](~/includes/deprecatedcontent-md.md)]  
+
+The wfc workflow command-line compiler tool works with old workflow markup file that has the file name extension .xoml (obsoleted)\\. 
+
+## Compilation Process
+
+When workflows are compiled, the following procedures are performed as part of the compilation process:
+
+  - Validation is performed to ensure that the workflow activities validate based on the rules that the activities have set for themselves. If there are validation errors, the compiler returns a list of the errors.  
+      
+  - A partial class is generated from the markup definition that is input into the compiler.  
+      
+  - Code is generated to help with the run-time execution of the activities. Event subscriptions are generated, which help activities know when the activities they contain are finished executing.  
+      
+  - The partial classes generated from the markup file and the partial classes from the code file are entered into the .NET Framework C\# or Visual Basic compiler. The output of this process is the .NET assembly, WorkflowSample.dll. This can be deployed to run the workflow.  
```

Comment:
```suggestion
- The partial classes generated from the markup file and the partial classes from the code file are entered into the .NET Framework C# or Visual Basic compiler. The output of this process is the .NET assembly, WorkflowSample.dll. This can be deployed to run the workflow.  
```